pkgs/os-specific/linux/raspberrypi-eeprom/default.nix

```diff
@@ -51,9 +51,9 @@ stdenvNoCC.mkDerivation rec {
   '';
 
   meta = with lib; {
-    description = "Installation scripts and binaries for the closed sourced Raspberry Pi 4 EEPROMs";
-    homepage = "https://www.raspberrypi.org/documentation/hardware/raspberrypi/booteeprom.md";
+    description = "Installation scripts and binaries for the closed sourced Raspberry Pi 4 and Raspberry Pi 5 bootloader EEPROMs";
```

Comment:
```suggestion
    description = "Installation scripts and binaries for the closed sourced Raspberry Pi 4 and 5 bootloader EEPROMs";
```

Reply:
Its common to be faithful to upstream, but some liberty for the sake of brevity or adding context/discoverability is always welcome. :)